app/controllers/signup_controller.rb

```diff
@@ -35,61 +30,68 @@ def signup_form
   def signup_post
     handle_with(
       SignupForm,
+      # TODO: what is all this stuff getting set for?
       contracts_required: !contracts_not_required,
       client_app: get_client_app,
       is_bri_book: is_bri_book_adopter?,
       success: lambda {
-        save_unverified_user(@handler_result.outputs.user.id)
```

Comment:
might need to make sure this isn't needed for something else - but I don't think it is